roles/ocm-attach/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Hub Cluster Tasks
+  include_tasks: hub_setup.yml
+  when:
+    - ocm_hub_kubeconfig is defined
+    - ocm_managedcluster_kubeconfig is not defined
+
+- name: Managed Cluster Tasks
+  include_tasks: managedcluster_setup.yml
+  when:
+    - ocm_hub_kubeconfig is defined
+    - ocm_managedcluster_kubeconfig is defined
```

Comment:
for the situation of 
```
    - ocm_hub_kubeconfig is defined
    - ocm_managedcluster_kubeconfig is defined 
 ```
 
hub_setup task still need to run or else the import secret will not be generated and `managedcluster_setup.yml` will fail

Reply:
in my opinion i think we should require that `ocm_managedcluster_kubeconfig` should exist for this task

Reply:
The hub_setup and the managedcluster_setup parts are now one set of tasks so all kubeconfigs will be required at run time. This ensures that there is always an import value ready.

However, this means that we need to conditionally check for already existing/registered managed clusters. And check that such clusters either might not be setup or have been setup. This was accounted for in PR.